tardis/utilities/executors/sshexecutor.py

```diff
@@ -175,6 +175,18 @@ async def run_command(self, command, stdin_input=None):
                     stderr="SSH Broken Connection",
                 ) from coe
             else:
+                # In case asyncssh loses the connection while running a command, the
+                # connection loss seems to be silently ignored, however the
+                # exit_status is None in that case.
+                if response.exit_status is None:
```

Comment:
In the case of `asyncssh.ChannelOpenError` we also clear to current connection to have it recreated. Is this useful here as well?

Reply:
@giffels Can you comment on this? As far as I can tell, the `self._ssh_connection` should be reset here as well considering that it's broken.